### ISSUE TITLE
Create 'Pro Add-ons' and 'Free Add-ons' tabs in Settings with all add-on settings

### ISSUE THREAD

**Issue by wordmaid** [AffiliateWP/AffiliateWP#1287]
I think it would be fantastic to streamline how add-ons are displayed in the admin Settings screen. Currently Recurring Referrals and Tiered Affiliate Rates add their own new tabs to the main Settings screen, but Lifetime Commissions goes into the Integrations tab below the integrations, and I believe one or two others add options to the Misc tab.

A **Pro Add-ons** tab that includes all pro add-on settings in the one tab would be awesome, and similarly a **Free Add-ons** tab that features all our free add-on settings in the one tab. Each add-on could be separated by a line similar to the style in the General tab. Even if there is just one checkbox for an add-on, it will make for a tidier and more seamless UX with everything in one place.

**Comment by sumobi** [comment]
👍  I think there should be 1 tab called "Add-ons". Both Pro and Free add-ons will be in there. 

Similar to how we output extension settings in EDD, we should create simple links along the top that a user can click on to reveal the settings for that particular add-on.

**Comment by wordmaid** [comment]
That could work too!

**Comment by pippinsplugins** [comment]
When we do this, let's make sure we add section support too, like we did in EDD: https://easydigitaldownloads.com/development/2016/01/07/updating-extension-settings-for-subsections-in-2-5/

**Comment by michaelbeil** [comment]
Yes, 100%. We were just talking about this in Slack the other day.

**Comment by michaelbeil** [comment]
Pardon the closure.

**Comment by pippinsplugins** [comment]
Moving this to 2.0.
